SasquatchMac/SasquatchMac/ViewControllers/AnalyticsViewController.swift

```diff
@@ -44,6 +46,7 @@ class AnalyticsViewController : NSViewController, MobileCenterProtocol {
     guard let `mobileCenter` = mobileCenter else {
       return
     }
+    ServiceStateStore.AnalyticsState = sender.state == 1
```

Comment:
We don't need this part I think. below line will update the value to storage since we are referencing the same key.